tests/switch/bench-switch.c

```diff
@@ -78,4 +79,20 @@ void test_benchmark_switch(int argc, char **argv)
 	pg_packets_free(bench.pkts, bench.pkts_mask);
 	pg_brick_destroy(sw);
 }
-
+void test_benchmark_switch(int argc, char **argv) 
+{
+	printf("\n");
+	printf("Test benchmarks with 20 edges at each sides \n");
```

Comment:
1: it's useless to use 2 printf for that, you can just prepend the "\n" at the begin of your second printf
2: printf like this in the bench are ugly and useless, as you can configure the title of the current bench in pg_bench_init, vtep bechs do that, and you can use them as examples.